mediaire_toolbox/queue/redis_wq.py

```diff
@@ -7,7 +7,7 @@
 import hashlib
 import logging
 
-logger = logging.getLogger(__name__)
+default_logger = logging.getLogger(__name__)
```

Comment:
why do we again need the default_logger?

Reply:
I just discover that my editor is renaming every ocurrence in all the projects, therefore this was also renamed, although it doesn't make sense. I will check and rename to logger.